src/test/components/__snapshots__/MarkerTooltipContents.test.js.snap

```diff
@@ -151,8 +149,7 @@ Array [
         <div
           className="tooltipTiming"
         >
-          0.80
-          ms
+          0.80ms
```

Comment:
Before the patch, I think we had 2 "text nodes" (maybe not on the HTML meaning but at least in the React meaning) and that's why it showed up like this. But I checked: we had no space. So I kept it the same.